setup.py

```diff
@@ -45,6 +49,7 @@ def _main():
     _check_dependency("skbuild", "scikit-build")
     _check_dependency("cmake")
     _check_dependency("ninja")
+    # _check_dependency("conan") # Can't check, because "pip install conan" does not install it as a python module.
```

Comment:
Did you try `_check_dependency("conan", "conans")`?

Reply:
Not sure how setting "package_hint" will change the outcome of 
`if module_name not in [module[1] for module in iter_modules()]`

Reply:
But aha, `import conans` works. Thanks. I'll try to extend the test so it works :)